docs/promtail.md

```diff
@@ -0,0 +1,71 @@
+## Promtail and scrape_configs
+
+Promtail is an agent which reads the Kubernetes pod log files and sends streams of log data to
```

Comment:
not only k8s, for all logs that promtail can get

Reply:
Good point, clarified.